sql/service_wsrep.cc

```diff
@@ -288,6 +288,7 @@ extern "C" bool wsrep_thd_set_wsrep_aborter(THD *bf_thd, THD *victim_thd)
     return true;
   }
   victim_thd->wsrep_aborter = bf_thd->thread_id();
+  DBUG_ASSERT(bf_thd->thread_id() != 0);
```

Comment:
Upstream has removed support for DBUG_ASSERT in 8.0.25 by https://github.com/percona/percona-server/commit/6889e5c00a51f6d8f9c79def33f6d55226f31f9a

Can you change it to just `assert()` so that there will be one less place to change when we merge 8.0.25?

```suggestion
  assert(bf_thd->thread_id() != 0);
```